### ISSUE TITLE
User's "Like"s to comments not shown

### ISSUE THREAD

**Issue by ShamrockLee** [AllanWang/Frost-for-Facebook#1590]
I am able to "like" or send an emotion to a comment of a post, which can be shown on a list offl peaple who do so. But it doesn't show at the left of the word "Like", making it hard for me to check whether i have "like"d it or which emoji I have send.

![obscura1575489128160_2](https://user-images.githubusercontent.com/44064051/70204844-f8fd4780-175c-11ea-8b76-14b2bbc763b9.jpg)

OS Version: 4.9.112-perf+ (1572277699)
OS SDK: 28
Device (Manufacturer): 
Model (Product): 
Package Installer: org.fdroid.fdroid
Tablet: false
Screen Dimensions:

App: com.pitchedapps.frost
App Version Name: 2.4.1
App Version Code: 2040100

Prev version: -1
Random Frost ID: 1575482313436-55014219-FO
Locale: 中文 (台灣)

**Comment by AllanWang** [comment]
Can you confirm that the same problem happens in your browser at m.facebook.com? I have seen reports that the like word and color doesn't change, but it seems to differ across the globe, and wasn't something I could replicate last time. Will take a look in the near future when I can. If you are familiar with development, consider sending me the relevant css through your browser so I can add it to the theme.

**Comment by ShamrockLee** [comment]
It works on m.facebook.com but not on the app. The "complete" copy of webpage is in the zip file, but there are only an HTML file and a folder of icons and no CSS files.
The result of "Inspect Element" in Firefox is shown below:
![m_facebook_com_1](https://user-images.githubusercontent.com/44064051/70906661-dc58ed80-2041-11ea-8d89-7ff053f4adb6.png)
![m_facebook_com_2](https://user-images.githubusercontent.com/44064051/70906692-f1358100-2041-11ea-93ca-70e81a38d329.png)

**Comment by AllanWang** [comment]
Apologies, it should be touch.facebook.com. I do use m internally but for
some locales or devices it becomes the really old site. Please also remove
your zip and email it to me, unless you are okay with pii. To save the css,
download the full page. As long as you can open the page and it looks the
same, then it's fine

**Comment by ShamrockLee** [comment]
Oops! Thanks a million.

**Comment by ShamrockLee** [comment]
@AllanWang My mail seems to be "blocked by the mail filter".
```
An error occurred while sending mail. The mail server responded:  5.7.0 This message was blocked because its content presents a potential
5.7.0 security issue. Please visit
5.7.0  https://support.google.com/mail/?p=BlockedMessage to review our
552 5.7.0 message content and attachment content guidelines. ****************** - gsmtp. Please check the message and try again.
```

**Comment by AllanWang** [comment]
@ShamrockLee this is because uploading js is prohibited. You have a few options. If drive allows it, zip your content and share a link to me. You can optionally rename all js files to `js.txt` (use a script if possible). I do have a debugging tool that does that and renames all the relevant usages in html, but I haven't used it in a while and haven't gotten to verifying it yet

**Comment by AllanWang** [comment]
Should be fixed in #1602
